### PR TITLE
forbid access to certain endpoints for Kafka deploy

### PR DESCRIPTION
**Pull request by tpboudreau-rackspace**

### CODE REVIEW

**Comment by tpboudreau-rackspace** [comment]
The filter scoping regex at https://github.com/mmi-cookbooks/ele-repose/pull/53/files#diff-25e5d4a4446ae12a0d6f1162b6160375R132 is just a placeholder for now.  Real one TBD.

File: templates/default/simple-rbac.cfg.xml.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<simple-rbac xmlns="http://docs.openrepose.org/repose/simple-rbac/v1.0" roles-header-name="X-ROLES">
+    <resources>
+/v1.0/{accountId}/entities/{entityId}/alarms/{alarmId}   GET             ALL
+/v1.0/{accountId}/entities/{entityId}/alarms/{alarmId}   PUT,DELETE      NOBODY
```
Comment (jjbuchan) [inline templates/default/simple-rbac.cfg.xml.erb]: Is this something that has to be set or is it optional?  Wondering what the default permissions are if the method is not specified.

**Comment by jjbuchan** [comment]
+1

File: templates/default/simple-rbac.cfg.xml.erb (hunk 1 above)
Comment (tpboudreau-rackspace) [inline templates/default/simple-rbac.cfg.xml.erb]: It depends on another setting, the mask-rax-roles-403 attribute.  As currently set, i think you'd get a  403; but you can reset that attribute to issue 405's if a method isn't specified for a covered endpoint.  I fiddled with it a bit -- but it didn't seem to be working exactly as advertised, so I went with what worked.

**Comment by jjbuchan** [comment]
Testing ghpw +1

**Comment by jjbuchan** [comment]
+1

**Comment by jjbuchan** [comment]
This change is no longer needed.  Closing

cc @jguice

**Pull request closed without merging**